torch/csrc/generic/methods/Tensor.cwrap

```diff
@@ -761,6 +761,12 @@ PyObject * THPTensor_(stride)(PyObject *self, PyObject *args, PyObject *kwargs)
         - accreal start
         - accreal end
         - CONSTANT 1
+      - arguments:
```

Comment:
We need to add this to aten/csrc/ATen/Declarations.cwrap as well

Reply:
I added this in my latest commit.

> Can you make sure this works on Variable too?

I'm not sure what you mean by this. Could you elaborate?

Reply:
I missed that change. The contbuild is failing because of a syntax error in that file. You'll  also probably have to make the same change as you did in `generic/methods/Tensor.cwrap` because C++ doesn't support optional arguments before non-optional ones. (Instead it has to be listed as two functions as you do here.)

To test it locally, you can run `python setup.py build develop`